### PR TITLE
Compound validation rule support

### LINKED ISSUE #1817: Feature request: support for compound validation rule
For this snippet open api is successfully generated
```php
final class SomeWebRequest
{
    /**
     * @Assert\Type("numeric")
     * @Assert\NotBlank()
     * @Assert\Positive()
     */
    private $someParam;
}
```
but for this - it doesn't
```php
namespace App\Validator\Constraints;

use Symfony\Component\Validator\Constraints\Compound;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @Annotation
 */
class SomeValidationRule extends Compound
{
    protected function getConstraints(array $options): array
    {
        return [
            new Assert\Type('numeric'),
            new Assert\NotBlank(),
            new Assert\Positive(),
        ];
    }
}

namespace App\Controller;

use App\Validator\Constraints as AcmeAssert;

final class SomeWebRequest
{
    /**
     * @AcmeAssert\SomeValidationRule
     */
    private $someParam;
}
```

Is it possible to make support for compound validation rules? https://symfony.com/doc/current/reference/constraints/Compound.html

### PR DESCRIPTION
**Pull request by melihovv**
Resolves #1817

### CODE REVIEW

**Comment by melihovv** [comment]
Compound constrained was introduced only in symfony 5.1.
I will try to fix tests for lower symfony versions.

**Comment by melihovv** [comment]
@GuilhemN please, add approve to run github actions

**Comment by melihovv** [comment]
@GuilhemN can you please give approve one more time?
It is very inconvenient that I need to ping you every time when new commit is pushed 😞

**Comment by GuilhemN** [comment]
I think it will happen until I merge a first PR :/ don't think I can fix that

**Comment by melihovv** [comment]
@GuilhemN all tests have passed, PR is ready to review, please check it when you can
I don't know why, but I cannot assign you as reviewer in github ui

**Comment by kirya-dev** [comment]
can be great add support for it
```php
/**
* @Assert\All({
 *     @Assert\Type("int")
* })
*/ 
```

File: ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php [hunk 1]
```diff
@@ -50,67 +50,73 @@ public function updateProperty($reflection, OA\Property $property): void
...
-                    $property->minimum = (int) $innerAnnotation->value;
-                }
+                $this->processPropertyAnnotation($reflection, $property, $innerAnnotation);
             }
         }
     }
 
+    private function processPropertyAnnotation($reflection, OA\Property $property, $annotation)
+    {
+
```
Comment (kirya-dev) [inline ModelDescriber/Annotations/SymfonyConstraintAnnotationReader.php:60]: extra line

**Comment by GuilhemN** [comment]
Thanks a lot @melihovv!
@kirya-dev I don't have much time implement support for `@All`, but a contribution is welcome if you'd like :)

**Comment by melihovv** [comment]
@GuilhemN can you make a new release please?

**Review by firoj07 — COMMENTED** [review]
Thanks 😊

**Comment by GuilhemN** [comment]
@melihovv there you go https://github.com/nelmio/NelmioApiDocBundle/releases/tag/v4.4.0 ;)

**Pull request merged**